packages/firestore/test/lite/integration.test.ts

```diff
@@ -2038,3 +2045,142 @@
     });
   });
 });
+
```

Comment:
Please add these two tests:
1. AggregateQuery.query === the original query
2. AggregateQuerySnapshot.query === the aggregate query